PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,21 @@
 ## Description
 
-What was changed in this pull request?
+> What was changed in this pull request?
 
-Why is it necessary?
+...
```

Comment:
IMO, this is just another line I have to overwrite when creating a PR. Not strictly necessary. Up to you if you want to keep it.

Reply:
Removed 👍